ciongke/cmd/hook/main.go

```diff
@@ -53,6 +53,11 @@ var defaultJenkinsJobs = map[string][]JenkinsJob{
 			Context:   "Jenkins GKE smoke e2e",
 		},
 		{
+			Name:    "kubernetes-pull-gci-build-test-e2e-gce",
+			Trigger: regexp.MustCompile(`@k8s-bot (gci )?(gce )?(e2e )?test this`),
```

Comment:
Ok, I made the gci bit required for this and for the gci kubemark and federation as well.